mw-webapp/src/component/button/Button.module.scss

```diff
@@ -1,9 +1,19 @@
+@import "src/styles/variables.scss";
+
 .button {
-  width:  200px;
-  height: 50px;
+  width:  100px;
+  height: 40px;
   margin: 20px;
+  background: $color-btn;
+  border-radius: 12px;
+
```

Comment:
margin, padding, border-radius and others better to move out to global styles because usually it's the same in all project.

Reply:
Done